frameworks/helloworld/src/main/dist/pod-profile-mount-volume.yml

```
@@ -14,7 +14,12 @@ pods:
       profile:
         goal: RUNNING
         essential: false
-        cmd: df -t $VOLUME_PROFILE pod-container-path >> pod-container-path/output && df -t $VOLUME_PROFILE profile-container-path >> profile-container-path/output && sleep $SLEEP_DURATION
+        cmd: >
```

Should we start the command with `set -e -x` for easier debugging?
```suggestion
        cmd: >
            set -e -x
```

---

👍